### DISCUSSION TITLE
nuclei -validate failed

### DISCUSSION THREAD

**Discussion by colaoo123** [projectdiscovery/nuclei#4733] [Q&A] [answered]
![1707149165401](https://github.com/projectdiscovery/nuclei/assets/136353760/d4934a83-1be4-42fa-8872-e044de3ddbba)

**Comment by colaoo123** [comment]
and also -t  
 it shows that upload to cloud failed and the scan failed with no result returned
![1707149563120](https://github.com/projectdiscovery/nuclei/assets/136353760/f11ea36a-aeea-4d08-be7a-d3a24a5c75ba)

**Comment by geeknik** [comment, marked as answer]
In reference to your first message, `sh, bash` don't exist on Windows, so the template will fail unless you've got `sh` or `bash` in your Windows path somewhere, but that's doubtful. Besides, running Linux privilege escalation templates on Windows is pointless.

In reference to your second message, you'll need to first run `nuclei -auth` and enter your PDCP API key. Then you'll need to use the command line flag `-cup` with your scan in order to upload the results.

**Comment by colaoo123** [comment]
thanks for your quick reply
one thing made me confused is that i scaned about 4k ip but no successful results returned
just like this but i think it should't have been 
![1707151794639](https://github.com/projectdiscovery/nuclei/assets/136353760/1d68ca80-362c-40a9-9641-d877b4f3e7c9)

**Comment by geeknik** [comment]
I would first upgrade your nuclei installation using `nuclei -up`. You're running v2.9.8 and v3.1.10 is the latest version. That might help with some of these other issues you're experiencing.

**Comment by colaoo123** [comment]
ok i'v updated  

last time i scaned 4k ip and got the same result like the photo shows （by the way the yaml is the same and right
so i cant figure out what was wrong now

![1707152807801](https://github.com/projectdiscovery/nuclei/assets/136353760/47927b82-0a9c-4a79-b95c-e4ff8d9bf9e4)

**Comment by geeknik** [comment]
To me, that looks like a successful scan where none of the 145 targets are vulnerable to the CVE you chose. 

I've tested the template out locally against known vulnerable Confluence hosts and the template appears to work as intended.

**Comment by colaoo123** [comment]
ok thanks dude ，great help truly

**Reply by geeknik** [reply]
You're welcome. Come back for help anytime!
